client/src/Components/Header/Notification/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Dropdown } from 'react-bootstrap';
+import './style.css';
+
+class Notification extends Component {
+  // this notification will fetched from back-end socket
+  state = {
```

Comment:
to be more clearly, add utilise file for notifications and import it here then in the ```componentDidMount``` function update the state with notification details.
this way will facilitate for us when joining the backend with front-end

Reply:
I think, we will change it when we use actual data from socket
thanks